primitives/python/qrmi_primitives/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we'll have to restructure these files when we put our Pasqal plugins in but I think we can merge this first and do that when relevant.